src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -677,13 +677,27 @@ s32 func_80997A34(DoorShutter* this, GlobalContext* globalCtx) {
 void DoorShutter_Draw(Actor* thisx, GlobalContext* globalCtx) {
     DoorShutter* this = THIS;
 
+//! @bug This actor is not fully initialized until the required object dependency is loaded.
+//! In most cases, the check for objBankIndex to equal requiredObjBankIndex prevents the actor
+//! from drawing until initialization is complete. However if the required object is the same as the
+//! object dependency listed in init vars (gameplay_keep in this case), the check will pass even though
+//! initialization has not completed. When this happens, it will try to draw the display list of the 
+//! first entry in `D_80998134`, which will likely crash the game.
+//! This only matters in very specific scenarios, when the door is unculled on the first possible frame
+//! after spawning. It will try to draw without having run update yet.
+//!
+//! The best way to fix this issue (and what was done in Majora's Mask) is to null out the draw function in
+//! the init vars for the actor, and only set draw after initialization is complete.
```

Comment:
Shouldn't these comments be indented? And I would be surprised if clang-format leaves it like that

Reply:
oh, i used githubs built in commit thing for mzx's suggestion lol. will fix in a moment